doc/user/assets/sass/_content.scss

```diff
@@ -937,6 +937,116 @@ p+p {
     }
 }
 
+div.json_widget {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    position: relative;
+
+    .input_container {
+        display: inline-flex;
+        flex-direction: row;
+
+        margin: 16px;
+
+        .input_container-text {
```

Comment:
Personal preference here, but I find that nesting selectors more than 1 or 2 level tends to make it harder to change resuse styles as well as evolve your design. That said, this sort of nesting appears to be conventional here, so it me best to be consistent.

Reply:
Yeah I didn't know what our preference was here, so I tried to just stick with what already existed